modules/monitoring-getting-detailed-information-about-a-target.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-targets.adoc
+
+[id="getting-detailed-information-about-a-target_{context}"]
+= Getting detailed information about a target
+
+The Targeting UI provides detailed information about HTTP endpoints that are targeted for metrics scraping.
+
+.Prerequisites
+
+* You have access to the cluster as an administrator for the project for which you want to view metrics targets.
+
+.Procedure
+
+*To view detailed information about a target in the Administrator perspective*:
+
+. Open the {product-title} web console and navigate to the *Observe* -> *Targets* -> *Metrics Targets* page.
+
+. Optional: Search for a target by name using the *Text* field in the search list.
+
+. Optional: Search for a target by label using the *Label* field in the search list.
+
+. Optional: Filter the targets by status and source by selecting filters in the *Filter* list.
```

Comment:
Would it make sense to combine these into a single point like `Optional: Filter the targets by name, label, status or source as described above` to avoid repetition? (Or just `Optional: Filter the targets as described above`?)

Reply:
I know that it can sound repetitive, but for the sake of modularization, I think it's better not to refer to "above" or "below" in a doc. If a user comes directly to this section, then it should be understandable without the user having to refer elsewhere in the content.

Reply:
👍 Thanks for the explanation